profile_bluesky/startup/instrument/devices/util_components.py

```diff
@@ -0,0 +1,155 @@
+
+"""
+Utilities
+"""
+
+from ophyd import (
+    Component, FormattedComponent, Signal, EpicsSignal, EpicsSignalRO,
+    PVPositioner
+)
+from ..session_logs import logger
+logger.info(__file__)
+
+
+class DoneSignal(Signal):
+    """ Signal that tracks if two values become the same. """
+    def __init__(self, *args, readback_attr='readback',
+                 setpoint_attr='setpoint', tolerance_attr='tolerance',
+                 **kwargs):
+        super().__init__(*args, **kwargs)
+        self._readback_attr = readback_attr
+        self._setpoint_attr = setpoint_attr
+        self._tolerance_attr = tolerance_attr
+
+    def get(self, **kwargs):
+        readback = getattr(self.parent, self._readback_attr)
```

Comment:
@qzhang234 , @gfabbris : Are the attribute names expected to change after an instance of this class is created?  If not, then the `getattr()` can be moved into the `__init__()` so they are done only once.

Reply:
Yes, all these `getattr()` should be in the `__init__()`. But these two utils (`DoneSignal` and `TrackingSignal`) is outdated (I will remove from the 4-ID-D setup), and I think it's not used anywhere here, so these could just be removed.